pages/index.js

```diff
@@ -7,7 +7,6 @@ export default function Home() {
 
 	return (
 		<>
-			<h1>caars</h1>
 			<AddButton onClick={() => setShow(true)}>+</AddButton>
```

Comment:
The Button-component should rather export one single Button with variants than different Button-types by themselves.

Reply:
adjusted and done.